docs/devel/README.md

```diff
@@ -199,6 +199,9 @@ below script. It will build the release tar push it to docker locally and run th
 The ```'kubeconfig'``` path should be ```$HOME/.kube/config```.
 The below script run based on the above assumptions.
 And it should point to a running k8s cluster.
+If your running k8s cluster was started by local-up-cluster.sh, you should ```export HOSTNAME_OVERRIDE=$master-ip``` before running local-up-cluster.sh.
+```$master-ip``` is the IP of your machine where will run k8s cluster.

```

Comment:
nit: ... is the non-loopback IP of your machine where running the k8s cluster.

Reply:
Done